packages/components/src/templates/next/components/internal/Navbar/MobileNavMenu/MobileNavItemAccordion.tsx

```diff
@@ -85,10 +81,10 @@ const ParentItemLink = ({
     >
       <Link
         LinkComponent={LinkComponent}
-        isExternal={isExternalUrl(referenceLinkHref)}
-        showExternalIcon={isExternalUrl(referenceLinkHref)}
+        isExternal={isExternalUrl(url)}
+        showExternalIcon={isExternalUrl(url)}
```

Comment:
nit: just realized maybe we can just use the `isExternal` props that's passed in?